packages/workspace/src/generators/ci-workflow/ci-workflow.ts

```diff
@@ -30,11 +29,19 @@ export async function ciWorkflowGenerator(host: Tree, schema: Schema) {
   if (!nxCloudUsed) {
     throw new Error('This workspace is not connected to Nx Cloud.');
   }
+  const packageJson = readJson(host, 'package.json');
+  const allDependencies = {
+    ...packageJson.dependencies,
+    ...packageJson.devDependencies,
+  };
+  const hasE2E =
+    allDependencies['@nrwl/cypress'] || allDependencies['@nrwl/playwright'];
```

Comment:
```suggestion
    allDependencies['@nx/cypress'] || allDependencies['@nx/playwright'];
```

Reply:
Good catch. That was copilot being smart and I missed it.